AndroidSDKCore/src/main/java/com/leanplum/internal/FeatureFlagManager.java

```diff
@@ -10,8 +10,6 @@
 public class FeatureFlagManager {
```

Comment:
Now, that we are accessing FeatureFlags directly , do we still need this class? FeatureFlagManager?

Reply:
FeatureFlagManager is the code that actually parses the response and keeps track of which flags are enabled. The FeatureFlag class is just where we keep track of the actual feature flag strings.
Changing where the constant strings are stored does not obviate the logic that parses the backend response and performs logic to check against it.
Is refactoring the strings to the different file somehow communicating that we no longer need the logic? Is there a better way that you have in mind that we can implement?

Reply:
👍